src/platform/packages/shared/kbn-esql-ast/src/commands_registry/types.ts

```diff
@@ -71,6 +71,7 @@ export interface ISuggestionItem {
   command?: {
     title: string;
     id: string;
+    arguments?: any[];
```

Comment:
We are avoiding anys. It must be a string[], no?
```suggestion
    arguments?: string[];
```